public/app/features/live/data/utils.ts

```diff
@@ -0,0 +1,55 @@
+import { DataQueryResponseData, isDataFrame } from "@grafana/data";
+import { StreamingDataFrame } from "./StreamingDataFrame";
+
+/**
+ * @alpha -- experimental
+ */
+ export enum StreamingResponseDataType {
```

Comment:
I really like that we moved this to core if it is not needed to be exposed to external projects!